src/ripple/protocol/impl/PublicKey.cpp

```diff
@@ -106,7 +106,7 @@ sliceToHex(Slice const& slice)
     @param strict_param whether to enforce strictly canonical semantics
 
     @note For more details please see:
-    https://ripple.com/wiki/Transaction_Malleability
+    https://xrpl.org/transaction-malleability.html#transaction-malleability
```

Comment:
This link will be a little more stable if you don't include the anchor.

```suggestion
    https://xrpl.org/transaction-malleability.html
```